Security groups fail in etcd.

On my test install, everything goes wrong. iptables makes it clear that parsing the rules hasn't gone well:

```
Chain felix-p-_1f35a710c69e75a-i (1 references)
 pkts bytes target     prot opt in     out     source               destination 
    0     0 DROP       all  --  any    any     anywhere             anywhere             /* ERROR failed to parse rules DROP: */

Chain felix-p-_1f35a710c69e75a-o (1 references)
 pkts bytes target     prot opt in     out     source               destination 
    0     0 DROP       all  --  any    any     anywhere             anywhere             /* ERROR failed to parse rules DROP: */
```

With lots of this in the logs:

```
2015-04-02 16:08:13,030 [ERROR][25] calico.felix.frules 158: Failed to convert rules to fragments: [{'ip_version': 4, 'dst_net': u'0.0.0.0/0', 'dst_tag': None, 'dst_ports': None}, {'ip_version': 6, 'dst_net': u'::/0', 'dst_tag': None, 'dst_ports': None}].  Will DROP!
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/calico-0.14-py2.7.egg/calico/felix/frules.py", line 152, in rules_to_chain_rewrite_lines
    on_deny=on_deny))
  File "/usr/local/lib/python2.7/dist-packages/calico-0.14-py2.7.egg/calico/felix/frules.py", line 187, in rule_to_iptables_fragment
    assert not unknown_keys, "Unknown keys: %s" % ", ".join(unknown_keys)
AssertionError: Unknown keys: ip_version
```

Here's an example of some rules:

```
ubuntu@calico-st02:~/calico$ etcdctl get /calico/policy/profile/20834313-58bc-4867-80c9-876c4c0f8ffa_7184c0df-294c-4932-8504-e5e1dd82b0cc/rules | python -m json.tool
{
    "inbound_rules": [
        {
            "icmp_type": null,
            "ip_version": 4,
            "protocol": "icmp",
            "src_net": null,
            "src_ports": null,
            "src_tag": "20834313-58bc-4867-80c9-876c4c0f8ffa"
        },
        {
            "ip_version": 4,
            "protocol": "tcp",
            "src_net": "172.18.203.223/32",
            "src_ports": [
                22
            ],
            "src_tag": null
        },
        {
            "ip_version": 6,
            "src_net": null,
            "src_ports": null,
            "src_tag": "7184c0df-294c-4932-8504-e5e1dd82b0cc"
        },
        {
            "ip_version": 4,
            "src_net": null,
            "src_ports": null,
            "src_tag": "7184c0df-294c-4932-8504-e5e1dd82b0cc"
        },
        {
            "ip_version": 4,
            "protocol": "tcp",
            "src_net": "172.18.203.223/32",
            "src_ports": [
                22
            ],
            "src_tag": null
        }
    ],
    "outbound_rules": [
        {
            "dst_net": "0.0.0.0/0",
            "dst_ports": null,
            "dst_tag": null,
            "ip_version": 4
        },
        {
            "dst_net": "::/0",
            "dst_ports": null,
            "dst_tag": null,
            "ip_version": 6
        },
        {
            "dst_net": "::/0",
            "dst_ports": null,
            "dst_tag": null,
            "ip_version": 6
        },
        {
            "dst_net": "0.0.0.0/0",
            "dst_ports": null,
            "dst_tag": null,
            "ip_version": 4
        }
    ]
}

```

--- a/calico/felix/frules.py
+++ b/calico/felix/frules.py
@@ -48,6 +48,7 @@ KNOWN_RULE_KEYS = set([
     "dst_tag",
     "dst_ports",
     "icmp_type",
+    "ip_version",
 ])
 
 

